commitment/commitment_test.go

```diff
@@ -527,9 +576,7 @@ func TestSplitCommitment(t *testing.T) {
 		{
 			name: "single input split commitment",
 			f: func() (*asset.Asset, *SplitLocator, []*SplitLocator) {
-				input := randAsset(
```

Comment:
Hmm, perhaps we still want to retain the randomization here w.r.t the actual asset details? IIUC the new dep here is that for everything to validate, we need to create thing in the following order: genesis, group key, asset.

Reply:
Yup I think this is a leftover from debugging before we removed witness validation from commitment tree construction.